app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -35,6 +37,42 @@ def events
       respond_for_index :collection => collection
     end
 
+    api :POST, "/hosts/subscriptions/register", N_("Simulate a host subscription-manager registration")
```

Comment:
Host create does not go through any of the dynflow actions, so we can't
easily handle backend actions for it.  If we want to restructure it to
potentially do that, thats fine.  I'll just remove system create here and
we can do that in the future.

On Thu, Mar 3, 2016 at 9:24 AM, Eric D Helms <notifications@github.com>
wrote:

> In app/controllers/katello/api/v2/host_subscriptions_controller.rb
> <https://github.com/Katello/katello/pull/5831#discussion_r54885542>:
>
> > @@ -35,6 +37,42 @@ def events
> >        respond_for_index :collection => collection
> >      end
> >
> > +    api :POST, "/hosts/subscriptions/register", N_("Simulate a host subscription-manager registration")
>
> If that is the case then I am losing track of why we need this special
> create when there is a host create already? Are we saying that each facet
> should provide an API method to create a host of that facet type?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello/pull/5831/files#r54885542>.
>